### PR TITLE
feat(port): add termux

### PR DESCRIPTION
**Pull request by holiq**
#### Description

This PR: implementation on Termux

#### Assets

Icon Termux
Termux Preview

#### Checklist

- [x] PR description included
- [x] I've read the [documents](https://github.com/daltonmenezes/aura-theme#documentation)
- [x] I've created or commented in an issue related to this port asking to work on it (#161)
- [x] I know I shouldn't change any files in the packages folder manually
- [x] I've added this port at the end of the related category list in the [main README](https://github.com/daltonmenezes/aura-theme/blob/main/README.md)
- [x] I've attached an image to the icon of the app that this port is related to
- [x] I've attached a screenshot with a good zoom in fullscreen showing this port in action

### CODE REVIEW

File: src/ports/termux/templates/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      {{{ author-thead }}}
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      {{{ author-tbody }}}
+    </tr>
+  </tbody>
+</table>
```
Comment (daltonmenezes) [inline src/ports/termux/templates/README.md:23]: Please, update it to:
```ts
# Contributors
<table>
  <thead>
    <tr>
      <td valign="bottom"><p align="center">
        <a href="https://github.com/holiq">
          <img src="https://github.com/holiq.png?size=100" align="center" />
        </a>
      </p></td>
      {{{ author-thead }}}
    </tr>
  </thead>

  <tbody>
    <tr>
      <td><a href="https://github.com/holiq">Holiq Ibrahim</a></td>
      {{{ author-tbody }}}
    </tr>
  </tbody>
</table>
```

File: src/ports/termux/templates/aura-theme.properties [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+color10=#54C59F
+color11=#C7A06F
+color12=#8464C6
+color13=#C17AC8
+color14=#6CB2C7
+color15=#DBDBDB
+background=#15141B
+foreground=#EDECEE
+cursor=#DBDBDB
+
```
Comment (daltonmenezes) [inline src/ports/termux/templates/aura-theme.properties:20]: We should not use hard coded color hashes, only tokens. Update it to:
```ts
color0={{ accent12 }}
color1={{ accent5 }}
color2={{ accent2 }}
color3={{ accent3 }}
color4={{ accent1 }}
color5={{ accent6 }}
color6={{ accent32 }}
color7={{ accent7 }}
color8={{ accent8 }}
color9={{ accent5Soft }}
color10={{ accent2Soft }}
color11={{ accent3Soft }}
color12={{ accent1Soft }}
color13={{ accent6Soft }}
color14={{ accent32Soft }}
color15={{ accent7Soft }}
background={{ accent12 }}
foreground={{ accent7 }}
cursor={{ accent7Soft }}
```

File: src/ports/termux/index.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  await createReadme({
+    template: resolve(templateFolder, 'README.md'),
+    replacements: {
+      portName,
+      version,
+      previewURL,
+    },
+  })
+}
```
Comment (daltonmenezes) [inline src/ports/termux/index.ts:29]: Your port uses dark and soft colors, you must pass both colorSchemes but to make this working in the same file, you have to rename the object keys from colorSchemes.darkSoft. Update the code to:
```ts
import { AuraAPI } from 'core'
import { resolve } from 'path'

export async function TermuxPort(Aura: AuraAPI) {
  const { createPort, createReadme, colorSchemes, constants } = Aura
  const templateFolder = resolve(__dirname, 'templates')
  const { info } = constants

  const portName = 'Termux'
  const version = '1.0.0'
  const previewURL = `https://github.com/${info.author.username}/assets/blob/master/images/${info.slug}/aura-termux-preview.png?raw=true`

  await createPort({
    template: resolve(templateFolder, `${info.slug}.properties`),
    replacements: {
      ...colorSchemes.dark,
      ...withDarkSoftScheme(colorSchemes),
      ...info,
    },
  })

  await createReadme({
    template: resolve(templateFolder, 'README.md'),
    replacements: {
      portName,
      version,
      previewURL,
    },
  })
}

function withDarkSoftScheme(colorSchemes: AuraAPI['colorSchemes']) {
  return Object.entries(colorSchemes.darkSoft).reduce(
    (acc, [key, value]) => {
      return {
        ...acc,
        [`${key}Soft`]: value,
      }
    },
    {}
  )
}
```

**Review by daltonmenezes — CHANGES_REQUESTED** [review]
Once you've made the requested changes, don't forget to rebuild the port and update the PR with those changes. Thanks

**Comment by holiq** [comment]
Okay i'll update it

File: src/ports/termux/index.ts (hunk 3 above)
Comment (holiq) [reply src/ports/termux/index.ts]: All done

**Review by daltonmenezes — APPROVED** [review]

**Comment by daltonmenezes** [comment]
Thanks @holiq 💜

**Comment by holiq** [comment]
I also thank you for merging my pr

**Pull request merged**